extensions/adobe/experience/adcloud/attributedconversions.schema.json

```diff
@@ -0,0 +1,241 @@
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "Post Attributed Conversion Events. Conversion Path can contain at the most ten events that contributed to the conversion event. ",
+  "definitions": {
+    "attributedconversions": {
+      "properties": {
+        "xdm:triggerType": {
+          "title": "Trigger Event Type",
+          "type": "string",
+          "description:": "Trigger Type for an Event - examples include click, impression"
+        },
+        "xdm:pathPosition": {
+          "title": "Path Position",
+          "type": "string",
+          "description": "Path Position of a click/impression event that attributed to a conversion event"
+        },
+        "xdm:logId": {
+          "title": "Internal identifier for the batch",
+          "type": "number",
+          "description": "Internal identifier for the batch represented in the following format of YYYYMMDDHH24"
+        },
+        "xdm:conversionTime": {
+          "title": "Conversion Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Conversion Time of the Event"
+        },
+        "xdm:triggerTime": {
+          "title": "Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Trigger Time of the Event"
+        },
+        "xdm:originalTriggerTime": {
+          "title": "Original Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Original Trigger Time of the Event"
+        },
+        "xdm:advertiserId": {
+          "title": "Advertiser Identifier",
+          "type": "string",
+          "description": "Identifier for the advertiser"
+        },
+        "xdm:searchEngineId": {
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name"
+        },
+        "xdm:campaignId": {
+          "title": "Campaign Identifier",
+          "type": "string",
+          "description": "Campaign Identifer on the advertising platform"
+        },
+        "xdm:placementId": {
+          "title": "Placement Identifier",
+          "type": "string",
+          "description": "Placement ID on the display advertising platform. It will be populated only for non-search advertising related campaigns"
+        },
+        "xdm:adgroupId": {
+          "title": "Ad Group Identifier",
+          "type": "string",
+          "description": "Ad Group ID on the search advertising platform"
+        },
+        "xdm:adId": {
+          "title": "Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the advertising platform"
+        },
+        "xdm:term": {
+          "title": "Search Term",
+          "type": "string",
+          "description": "Search Term that contributed to the conversion"
+        },
+        "xdm:termId": {
```

Comment:
Same comment as in `adconversions.schema.json`. Can you use `searchTermId` instead to be more explicit.